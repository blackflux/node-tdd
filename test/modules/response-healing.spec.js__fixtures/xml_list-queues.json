{
  "fn": "call",
  "params": [
    "sqs:ListQueuesCommand",
    {
      "QueueNamePrefix": "queue-na"
    }
  ],
  "cassette": [
    {
      "scope": "https://sqs.us-west-2.amazonaws.com:443",
      "method": "POST",
      "path": "/",
      "body": "QueueNamePrefix=queue-na&MaxResults=1000&Action=ListQueues&Version=2012-11-05",
      "status": 200,
      "response": "<?xml version=\"1.0\"?><ListQueuesResponse xmlns=\"http://queue.amazonaws.com/doc/2012-11-05/\"><ListQueuesResult><QueueUrl>https://sqs.us-west-2.amazonaws.com/799715345196/queue-name</QueueUrl><NextToken>AAcAAgGAARSS5TwyARjKvEtzRiU9-7SLAAAAAQAET2RpbgAVMgAAAIAAAAAMADV49xkN9M81B1-uACCMSfA4lSikCuU6IfjOm61yC2XRJCgrqG454BMG9s91VQIAAAAADAAAEAAAAAAAAAAAAAAAAACuISguSDtCCtrLIMNdib1c_____wAAAAEAAAAAAAAAAAAAAAEAAACsum_OBChhP2KkJbd8aE8zIxNaJVkyY47PG8XJTtmTYDskzekNdLuLn54pq7hJY3V4xlpzIfVUzrn94lkVzN9M-IvkkJylyF44cVGgMUBQagjjI3v2oKy6aOenskJgvly264lHDjfuIh2XEdgdVW-lvcPZHauFtjY-JtXxIta9iLYDFOonRUWC45jnv6y4ayK8Eu7gtlWa-Jn_ORgsYo9y0tecQVhhlXEswB3M864NnyG6U-6spJQKjogOyB4=</NextToken></ListQueuesResult><ResponseMetadata><RequestId>93225a9a-fc00-5341-bce5-00e8c6330d6b</RequestId></ResponseMetadata></ListQueuesResponse>",
      "responseIsBinary": false
    }  ]
}
