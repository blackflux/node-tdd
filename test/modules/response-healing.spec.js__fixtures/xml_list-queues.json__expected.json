[
  {
    "scope": "https://sqs.us-west-2.amazonaws.com:443",
    "method": "POST",
    "path": "/",
    "body": {
      "QueueNamePrefix": "queue-na"
    },
    "status": 200,
    "response": {
      "QueueUrls": [
        "https://sqs.us-west-2.amazonaws.com/799715345196/queue-name"
      ],
      "NextToken": "AAcAAgGAARSS5TwyARjKvEtzRiU9-7SLAAAAAQAET2RpbgAVMgAAAIAAAAAMADV49xkN9M81B1-uACCMSfA4lSikCuU6IfjOm61yC2XRJCgrqG454BMG9s91VQIAAAAADAAAEAAAAAAAAAAAAAAAAACuISguSDtCCtrLIMNdib1c_____wAAAAEAAAAAAAAAAAAAAAEAAACsum_OBChhP2KkJbd8aE8zIxNaJVkyY47PG8XJTtmTYDskzekNdLuLn54pq7hJY3V4xlpzIfVUzrn94lkVzN9M-IvkkJylyF44cVGgMUBQagjjI3v2oKy6aOenskJgvly264lHDjfuIh2XEdgdVW-lvcPZHauFtjY-JtXxIta9iLYDFOonRUWC45jnv6y4ayK8Eu7gtlWa-Jn_ORgsYo9y0tecQVhhlXEswB3M864NnyG6U-6spJQKjogOyB4="
    },
    "responseIsBinary": false,
    "reqheaders": {
      "amz-sdk-invocation-id": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$",
      "amz-sdk-request": "attempt=1; max=3",
      "authorization": "^AWS4-HMAC-SHA256 Credential=.+$",
      "content-length": "^[1-9][0-9]*$",
      "content-type": "application/x-amz-json-1.0",
      "host": "sqs.us-west-2.amazonaws.com",
      "user-agent": "^aws-sdk-js/.+$",
      "x-amz-content-sha256": "^[a-f0-9]{64}$",
      "x-amz-date": "^[0-9]{8}T[0-9]{6}Z$",
      "x-amz-target": "AmazonSQS.ListQueues",
      "x-amz-user-agent": "^aws-sdk-js/[0-9]+.[0-9]+.[0-9]+$"
    }
  }
]
